pagerduty.tf

```diff
@@ -1,24 +1,118 @@
-resource "newrelic_alert_channel" "pagerduty" {
+locals {
+  summary       = "{{ annotations.title.[0] }}"
+  customDetails = <<-EOT
+    {
+      "condition_name": {{json accumulations.conditionName.[0]}},
+      "details": {{json issueTitle}},
+      "duration": {{#if issueDurationMs}}{{issueDurationMs}}{{else}}0{{/if}},
+      "event_type": "INCIDENT",
+      "incident_acknowledge_url": {{json issueAckUrl}},
+      "incident_url": {{json issuePageUrl}},
+      "policy_name": {{json accumulations.policyName.[0]}},
+      {{#if accumulations.runbookUrl}}"runbook_url": {{json accumulations.runbookUrl.[0]}},{{/if}}
```

Comment:
This field can be empty if not specified in the alert.  I believe (because documentation does not say otherwise) the rest of the fields are present.
This is the documentation by the way: https://docs.newrelic.com/docs/alerts-applied-intelligence/applied-intelligence/incident-workflows/incident-workflows/